features/src/asserts.py

```diff
@@ -12,11 +12,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Custom asserts."""
+"""Custom asserts that can be imported into other source files and steps definitions."""
 
 
 def assert_sets_equality(what, expected, actual):
-    """Compare two sets of values."""
+    """Compare two sets of values, displays correct set difference when inequal."""
     assert expected == actual, "Difference found in sets of {}: {}".format(
         what, expected ^ actual
     )
+
+
+if __name__ == "__main__":
```

Comment:
is this code intended or a leftover?

Reply:
the checks? it is intended. Ideally should be part of all scripts :)